.github/workflows/auto-label-issues.yml

```
@@ -42,6 +53,7 @@ jobs:
           LABELS: to-triage
         run: |
           echo "Applying triage label for new issue"
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] expecting a single ${{...}} expression or float number literal, but found plain text node

(syntax-check)